### PR TITLE
Create Profile Card to organization page

### PR DESCRIPTION
**Pull request by mateusoliveira2**
## Create profile card

#34 

## What is the purpose of this pull request?
same than the title

## Screenshots or example usage:
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: components/organization/ProfileCard.vue [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+</template>
+
+<script>
+export default {
+  name: 'ProfileCard',
+  props: ['profile']
+}
+</script>
+
+<style>
```
Comment (thayannevls) [inline components/organization/ProfileCard.vue]: style scoped

File: components/organization/ProfileCard.vue [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+              <div class="description">
+                <br /><span
+                  >aqui vai ter a descrição do user e tudo mais que der né</span
+                ><br />
+                <br />
+              </div>
+            </div>
+            <footer class="box-footer">
+              <table style="width:100%">
```
Comment (thayannevls) [inline components/organization/ProfileCard.vue]: Don't use `style` attribute, always define a class.

File: components/organization/ProfileCard.vue [hunk 3]
```diff
@@ -0,0 +1,93 @@
+<template>
+  <div>
+    <div id="app">
```
Comment (thayannevls) [inline components/organization/ProfileCard.vue]: Not a good id

File: components/organization/ProfileCard.vue [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  position: absolute;
+  align-self: center;
+  bottom: 0;
+}
+.card {
+  width: 250px;
+  height: 350px;
+  background-color: #47525e;
+  color: white;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
```
Comment (thayannevls) [inline components/organization/ProfileCard.vue]: There is a default font for all the application, don't define other

**Comment by mateusoliveira2** [comment]
Don't have the tests because i don't see what testing without fetch real data

**Pull request merged**